Description/tr-TR.md

```diff
@@ -0,0 +1,89 @@
+# Türkçe
```

Comment:
The title in the original source has country shortcode in a paranthesis, it should be like `Türkçe (TR)`

Reply:
Solved